Cargo.toml

```diff
@@ -5,7 +5,9 @@ authors = ["Richard Lupton <richard.lupton@gmail.com>"]
 edition = '2018'
 
 [dependencies]
-quicli = "0.3"
+quicli = "0.4"
```

Comment:
Not keeping this in alphabetical order, or anything?

Reply:
Probably a good idea - will fix.